src/choleskyinv.jl

```diff
@@ -0,0 +1,151 @@
+using LinearAlgebra
+
+import Base: show, iterate
+
+"""
+Cholesky factorization and inverse Cholesky factorization, accessible
+in fields `.c` ans `.ci`, respectively.
+"""
+struct Choleskyinv{T<:Factorization}

```

Comment:
This should be called `CholeskyInv` and be `<: Factorization`. The arguments could be more descriptive but I couldn't think of a better name. You might have to change it to:
```julia
struct CholeskyInv{T, F<:Factorization{T}} <: Factorization{T}
     c::F
     ci::F
end

Reply:
Thanks, your proposition seems to work fine